pkg/master/master.go

```diff
@@ -47,9 +46,7 @@ import (
 	extensionsapiv1beta1 "k8s.io/kubernetes/pkg/apis/extensions/v1beta1"
 	networkingapiv1 "k8s.io/kubernetes/pkg/apis/networking/v1"
 	policyapiv1beta1 "k8s.io/kubernetes/pkg/apis/policy/v1beta1"
-	rbacv1alpha1 "k8s.io/kubernetes/pkg/apis/rbac/v1alpha1"
```

Comment:
don't turn this off... it's been on since 1.5 and we need to ensure we can read existing data. new data is written in v1beta1 as of 1.6